drivers/SmartThings/sonos/src/init.lua

```diff
@@ -36,7 +38,13 @@ local function find_player_for_device(driver, device, should_continue)
     --- @param ssdp_group_info SonosSSDPInfo
     SSDP.search(SONOS_SSDP_SEARCH_TERM, function(ssdp_group_info)
       driver:handle_ssdp_discovery(ssdp_group_info, function(dni, _, _, _)
+        log.info(string.format(
```

Comment:
Logging this to hub logs could really help us during migration should we need to look into issues.

Reply:
We override info in this file to use info_with

Reply:
I missed that log was overridden, so this and many others are going to logs already.